### PR TITLE
Add self (Michael Robson) to metadata contribution list

### PR DESCRIPTION
**Pull request by mprobson**

### CODE REVIEW

**Comment by mprobson** [comment]
Apparently I need to list my "Conflicts of Interest" (COI), which is not listed in the original format.

**Comment by cgreene** [comment]
COI is specific to this repo so that we will have it as new folks contribute. Can you add it mirroring the format for the existing authors?

https://github.com/greenelab/covid19-review/blob/master/content/metadata.yaml

Thanks!

File: content/metadata.yaml [hunk 1]
```diff
@@ -40,3 +40,13 @@ authors:
...
       lastapproved: !!str 2020-03-22
     funders: the Gordon and Betty Moore Foundation (GBMF 4552)
+  -
+    github: mprobson
+    name: Michael P. Robson
+    initials: MPR
+    orcid: 0000-0002-4859-0033
+    email: michael.robson@villanova.edu
+    contributions:
+      - Document Cleanup
```
Comment (cgreene) [inline content/metadata.yaml]: Can you use one of the CRediT taxonomy roles? I'm imagining this is "Methology" "Validation" or "Software". @rando2 what do you think is the appropriate one?

These are the standard roles: https://casrai.org/credit/

**Comment by rando2** [comment]
I think Software makes the most sense! This process needs some clean up because the directions aren't clear right now re: COI, CRediT. I think @mprobson is opening an issue, or else I can.
________________________________

**Comment by mprobson** [comment]
- Taxonomy updated (Software)
- COI added

**Comment by mprobson** [comment]
> I think Software makes the most sense! This process needs some clean up because the directions aren't clear right now re: COI, CRediT. I think @mprobson is opening an issue, or else I can.

Opening an issue now.

**Comment by mprobson** [comment]
Thanks for all the helps/suggestions! I appreciate you welcoming a newcomer to this project.

**Comment by mprobson** [comment]
> > I think Software makes the most sense! This process needs some clean up because the directions aren't clear right now re: COI, CRediT. I think @mprobson is opening an issue, or else I can.
> 
> Opening an issue now.

Please see #18 for more detail.

**Pull request merged**